packages/chatbot-eval-mongodb-public/testCases/included_links_conversations.yml

```diff
@@ -0,0 +1,832 @@
+- name: Search for 'and'
+  tags:
+    - top_search_term
+  messages:
+    - role: user
+      content: and
+  expectation: "N/A"
```

Comment:
All these `expectation: "N/A"` make me think we should tweak the data model to make `expectation` optional at some level. Thoughts?

Reply:
yea we can do that. will need to touch some other files as well to take into account the optionality. PR will grow a bit.